app/resources/brave_generated_resources_en-GB.xtb

```diff
@@ -100,47 +100,47 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK"/>.</translation>
 <translation id="5039804452771397117">Allow</translation>
 <translation id="7053983685419859001">Block</translation>
 <translation id="1679356339067548863">Help improve Brave's product stability by automatically sending diagnostic reports when the Brave Browser crashes or freezes.</translation>
-<translation id="660171790038410175"><ph name="BROWSER_NAME"/> quit unexpectedly.</translation>
+<translation id="660171790038410175"><ph name="BROWSER_NAME"/> quit unexpectedly</translation>
```

Comment:
Any .xtb files are automatically generated and will be overwritten. Any changes to the translations would have to be made via our localization vendor.